bindings/c/fdb_c.cpp

```diff
@@ -362,6 +362,10 @@ extern "C" DLLEXPORT FDBFuture* fdb_database_create_snapshot(FDBDatabase* db,
 // A value of 0 indicates that the client is more or less idle
 // A value of 1 (or more) indicates that the client is saturated
 extern "C" DLLEXPORT double fdb_database_get_main_thread_busyness(FDBDatabase* d) {
+	if (g_api_version < 631) {
+		return 0;
```

Comment:
I'm not sure if there's something better we can do here to make this less usable at old API versions.

Reply:
More specifically, I meant the above comment assuming we didn't change the API. It would be easier if we did change the API (e.g. `fdb_error_t fdb_database_get_main_thread_busyness(FDBDatabase* d, double *outBusyness)`, but it would be nice if this particular backward compatibility measure didn't dictate what API we choose.